storage/gc_queue.go

```diff
@@ -236,27 +236,20 @@ func (gcq *gcQueue) process(now proto.Timestamp, repl *Replica) error {
 	wg.Wait()
 
 	// Resolve all intents.
-	// TODO(spencer): use a batch here when available.
+	var intents []proto.Intent
```

Comment:
Why create this duplicate slice? Just pass `intentMap[id]` to `repl.resolveIntents()`